tox.ini

```diff
@@ -62,6 +62,7 @@ commands =
 	pip install -e .
 	pip install -r requirements/tests.txt
 	flake8
+	pip install git+https://github.com/gridsingularity/gsy-framework.git@rename-bids-offers
```

Comment:
It's not too small a refactoring to skip ticket for reference. Okay for now but for future would suggest to track such refactoring via ticket too